src/get-started/npm-script.md

```diff
@@ -5,4 +5,31 @@ title: Get Started > NPM Script
 
 # NPM Script
 
-Need to write up.
+If you already have a build process based on npm scripts you can use `workbox-cli` to generate a service worker.
+
+1. [Install Node.js](https://nodejs.org/en/) and create a new project using `npm-init`.
+1. Install the module with NPM.
+
+    ```
+    npm install --save-dev sw-build
+    ```
+
+1. Add the following to your `package.json`:
+
+    ```
+	"scripts": {
+	    "build": "workbox-cli generate:sw"
+	},
+	```
+
+	Or add just the middle line if your `package.json` already has a scripts section.
```

Comment:
I would kind of merge this section and the previous section to:

In your package.json, add a swBuild script like so:

<scripts code sample>

This should be enough for someone in the know to get going.